suspect/fitting/singlet.py

```diff
@@ -205,35 +190,34 @@ def model_to_parameters(model_dict):
         # Calculate dependencies/references for each parameter.
         depend_dict = calculate_dependencies(model_dict)
 
+        model_dict_copy = copy.deepcopy(model_dict)
+        params.append(("phase0", model_dict_copy.pop("phase0")))
+        params.append(("phase1", model_dict_copy.pop("phase1")))
+
         # Construct lmfit Parameter input for each parameter.
-        for name1, value1 in model_dict.items():
-            if type(value1) is int:  # (e.g. phase0)
-                params.append((name1, value1))
-            if type(value1) is dict:
+        for peak_dictionary, peak_property in model_dict_copy.items():
```

Comment:
I don't like the use of peak_dictionary here as this variable is the dict key, which is a string containing the peak **name**. `peak_property` is also not quite right as it is a dictionary containing multiple properties, `peak_properties` would be better. Make that change and I will accept the PR